app/form/v1_0/fields/drug-use/drug-use-history.ts

```diff
@@ -82,9 +105,28 @@ const drugsAffectedTheirLife: FormWizard.Field = {
   labelClasses: utils.getMediumLabelClassFor(FieldType.CheckBox),
 }
 
-const drugsAffectedTheirLifeDetails = FieldsFactory.detailsField({
-  parentField: drugsAffectedTheirLife,
-})
+const drugsAffectedTheirLifeDetails: FormWizard.Field = {
+  text: 'Give details (optional)',
+  code: fieldCodeWith(drugsAffectedTheirLife.code, 'details'),
+  type: FieldType.TextArea,
+  validate: [
+    {
+      type: 'validateMaxLength',
+      fn: utils.validateMaxLength,
+      arguments: [characterLimits.default],
+      message: `Details must be ${characterLimits.default} characters or less`,
+    },
+  ],
+  transform(state): FormWizard.Field {
+    const subject = state.session.subjectDetails as HandoverSubject
+    return {
+      ...this,
+      summary: {
+        text: `Details on how ${subject.givenName}'s drug use has affected their life`,
```

Comment:
I don't see where this has come from in the ACs for ARN-2702 or the Figma. Can you point it out please?

Reply:
this was in a comment from Cal on 21 May